compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -135,10 +135,10 @@ static void addTileAndDistributePasses(
       createFoldAffineMinInDistributedLoopsPass());
   nestedModulePM.addPass(createCanonicalizerPass());
   nestedModulePM.addPass(createCSEPass());
-  if (clEnablePadConsumerFusion && useFuseTensorPadWithConsumerPass) {
-    nestedModulePM.addNestedPass<func::FuncOp>(
-        createFuseTensorPadWithConsumerPass());
-  }
+  nestedModulePM.addNestedPass<func::FuncOp>(
```

Comment:
Let's delete `useFuseTensorPadWithConsumerPass` from the function arguments. We no longer need it, nice!